### PR TITLE
Improved / added new spells

### PR DESCRIPTION
**Pull request by GameDungeon**
No current use of Explosive Overclock.

### CODE REVIEW

File: spells.txt [hunk 1]
```diff
@@ -79,3 +81,12 @@
...
   message = "enter berserk mode" "goes berserk"
 }
+
+#GD ----------------------------------------------------------------------------------------
+
+"Explosive Overclock"
+{
+symbol = "X"
+cooldown = 1
+effect = Filter AUTOMATON Chain { Area 5 Chain{Damage SPELL_DAMAGE SPELL Blast} Area 3 PlaceFurniture "FIRE_WALL" Suicide DIE}
```
Comment (LudwigVonChesterfield) [inline spells.txt]: this is a self-cast spell, please remove Filter AUTOMATON part

File: spells.txt [hunk 2]
```diff
@@ -27,11 +27,13 @@
 "advanced automata repair"
 {
   symbol = "🔧"
-  effect = Area 3 Filter AUTOMATON RegrowBodyPart 10
+  #GD: better healing / should make a main pull to keeperrl to add Automation as a flesh type.
+  effect = Area 3 Filter AUTOMATON Filter ALLY Chain{ RegrowBodyPart 10 Heal FLESH }
```
Comment (LudwigVonChesterfield) [inline spells.txt]: automata repair had no Heal FLESH part, so I do not think this one should neither

File: spells.txt (hunk 1 above)
Comment (GameDungeon) [reply spells.txt]: Fixed

File: spells.txt (hunk 2 above)
Comment (GameDungeon) [reply spells.txt]: Fixed

**Pull request merged**